### PR TITLE
feat(NET-1069): dashboard changes

### PR DESCRIPTION
**Pull request by pwilson77**

### CODE REVIEW

File: src/components/modals/add-enrollment-key-modal/AddEnrollmentKeyModal.tsx [hunk 1]
```diff
@@ -187,7 +187,12 @@ export default function AddEnrollmentKeyModal({
           <Row ref={keyNetworksSelectRef}>
             {' '}
             <Col xs={24}>
-              <Form.Item name="networks" label="Networks" data-nmui-intercom="add-enrollment-key-form_networks">
+              <Form.Item
+                name="networks"
+                label="Networks"
+                data-nmui-intercom="add-enrollment-key-form_networks"
+                initialValue={[networkId]}
```
Comment (Aceix) [inline src/components/modals/add-enrollment-key-modal/AddEnrollmentKeyModal.tsx:194]: check if networkId is present before assigning else we'll get that empty selection when creating a new enrollment key from enrollmentkeys page. `initialValue={networkId ? [networkId] : undefined}`

File: src/components/modals/add-network-modal/AddNetworkModal.tsx [hunk 2]
```diff
@@ -17,6 +17,8 @@ import {
   isValidIpv6Cidr,
 } from '@/utils/NetworkUtils';
 import { convertUiNetworkToNetworkPayload } from '@/utils/NetworkUtils';
+import { NetworkUsecaseString } from '@/store/networkusecase';
+import { networkUsecaseMapText } from '@/utils/Utils';
```
Comment (Aceix) [inline src/components/modals/add-network-modal/AddNetworkModal.tsx:21]: remove if unused

File: src/components/modals/add-network-modal/AddNetworkModal.tsx [hunk 3]
```diff
@@ -254,6 +265,37 @@ export default function AddNetworkModal({
...
+                      style={{ width: '100%' }}
+                      options={Object.keys(networkUsecaseMapText).map((key) => {
+                        return { label: networkUsecaseMapText[key as NetworkUsecaseString], value: key };
+                      })}
+                    ></Select>
+                  </Form.Item>
+                </Col>
+              </Row>
+            </Col>
+          </Row> */}
```
Comment (Aceix) [inline src/components/modals/add-network-modal/AddNetworkModal.tsx:297]: remove if unused

File: src/components/modals/quick-setup-modal/QuickSetupModal.tsx [hunk 4]
```diff
@@ -0,0 +1,701 @@
...
+  'networks',
+  // 'hosts',
+  'remote_access_gateways',
+  'users',
+  'egress',
+  'ranges',
+];
+
+const RAC_LINK = 'https://docs.netmaker.io/pro/rac.html';
+const WIREGUARD_LINK = 'https://docs.netmaker.io/integrating-non-native-devices.html';
```
Comment (Aceix) [inline src/components/modals/quick-setup-modal/QuickSetupModal.tsx:78]: subsequently, we should move all the docs and external links to either env vars or a constsnts file so refactoring is less error-prone. will make a ticket for this

File: src/components/modals/server-malfunction-modal/ServerMalfunctionModal.tsx [hunk 5]
```diff
@@ -153,7 +153,7 @@ export default function ServerMalfunctionModal({ isOpen, onCancel }: ServerMalfu
         algorithm: store.currentTheme === 'dark' ? theme.darkAlgorithm : theme.defaultAlgorithm,
       }}
     >
-      <Modal open={isOpen} onCancel={onCancel} footer={null} centered closable={false}>
+      <Modal open={false} onCancel={onCancel} footer={null} centered closable={false}>
```
Comment (Aceix) [inline src/components/modals/server-malfunction-modal/ServerMalfunctionModal.tsx:156]: why this change?

File: src/pages/DashboardPage.tsx [hunk 6]
```diff
@@ -121,24 +175,146 @@ export default function DashboardPage(props: PageProps) {
...
+                width: '100%',
+                height: '259px',
+              }}
+            >
+              <Carousel adaptiveHeight={false} autoplay autoplaySpeed={10000}>
+                <div>
+                  <Card
+                    style={{
+                      height: '259px',
+                      background: 'linear-gradient(90deg, #52379F 0%, #B66666 100%)',
```
Comment (Aceix) [inline src/pages/DashboardPage.tsx:205]: we should put the gradient on the new guide instead. we want users to focus on that now

Comment (Aceix) [reply src/pages/DashboardPage.tsx:205]: ..amd take it off the current one

**Review by Aceix — CHANGES_REQUESTED** [review]
the image assets should be moved to public so it works well on saas

File: src/components/modals/server-malfunction-modal/ServerMalfunctionModal.tsx (hunk 5 above)
Comment (pwilson77) [reply src/components/modals/server-malfunction-modal/ServerMalfunctionModal.tsx:156]: it slipped through, I think i was working with bad internet and i disabled the modal

Comment (pwilson77) [reply src/components/modals/server-malfunction-modal/ServerMalfunctionModal.tsx:156]: good catch

File: src/pages/DashboardPage.tsx (hunk 6 above)
Comment (pwilson77) [reply src/pages/DashboardPage.tsx:205]: oh alright

File: src/components/modals/add-enrollment-key-modal/AddEnrollmentKeyModal.tsx (hunk 1 above)
Comment (pwilson77) [reply src/components/modals/add-enrollment-key-modal/AddEnrollmentKeyModal.tsx]: updated

File: src/components/modals/add-network-modal/AddNetworkModal.tsx (hunk 3 above)
Comment (pwilson77) [reply src/components/modals/add-network-modal/AddNetworkModal.tsx]: removed
